framework/physics/physics_material/multi_group_xs/single_state_mgxs.h

```diff
@@ -80,6 +42,14 @@ class SingleStateMGXS : public MultiGroupXS
    */
   void MakeCombined(std::vector<std::pair<int, double>>& combinations);
 
+  /**
+   * Scale the cross sections by the specified factor.
+   *
+   * @note Scaling factors do not compound. Each time this routine is called, the cross sections
+   *       are scaled by the ratio of the argument and the existing scaling factor.
```

Comment:
so scaling is always, in the end, the multiplier you want.

if the current scaling was 2 (meaning a factor 2 had already been applied) and you now give 3, then the data is multiplied by 1.5, making it effectively a factor 3 since the beginning. Right?

Reply:
Exactly! This allows us to always have a mapping back to microscopic cross sections. The scaling factor is what gets applied to the _unscaled_ cross sections